qml/ThemeColors.qml

```diff
@@ -0,0 +1,42 @@
+/*
+ * <one line to give the program's name and a brief idea of what it does.>
+ * Copyright (C) 2018  <copyright holder> <email>
+ * 
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ * 
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ * 
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import QtQuick 2.0
+
+/**
+ * This Component is used to specify colors that depends on a theme but cannot be suppliy by it.
+ */
+
+QtObject {
+    property var themes : {
+		'Ubuntu.Components.Themes.Ambiance' : {
+			'leisure_time' : "#ffffff",
+			'business_time' : "#efefef"
+		},
+		'Ubuntu.Components.Themes.SuruDark' : {
```

Comment:
Instead of writing the theme names (which might change, or new themese might appear), I would prefer if the decision on the color is based on the qualities of the theme color. For example, here you could replace the theme names with just `"light"` and `"dark"`, and decide which one to use based on the brightness of the current theme's background color. Something like
```
if (Theme.palette.normal.background.hslLightness > 0.5) {
    theme = "light"
} else
    theme = "dark"
}
```
(I'm not sure if the value ranges from 0 to 1 or from 0 to 255, you should check).

Reply:
I'm lost with this theming color palette stuff... No clue where i got this color from actually - i was aware of the existing color palette and thought i had respected it but maybe i've decided that there is no appropriate color...
@dark-eye may you discuss that with @mardy please? :grin:

Reply:
@mardy I like this idea! i`l will implement it once i have time.
but this was a solution to specific issue that the background color wasn't noticeable enough so generalizing it  will just lead to the same issue (if we will have other themes in the future) in other words my plan is to probably change the design to *fallback* to the "light" / "dark" theme color and still use specific  theme name for the colors then we get the best of  the specific and the generalize solutions.

Reply:
Just in case comes handy, there's a tutorial about palette on the old Canonical page:
https://docs.ubuntu.com/phone/en/apps/qml/tutorials-ubuntu-ui-toolkit-palette

Reply:
@dark-eye you (should) have write access to the branch, so you could push your commits directly to the branch, but you may also create a pull requests, just as you like to have it :)